client/src/components/ElementEditor/Element.js

```diff
@@ -52,17 +74,12 @@ const Element = (props) => {
         captureDraggingState: true,
       });
     }
-    // Check if formSchema state has already been loaded before opening a block
-    // This can happen if there was a validation error on a block after performing a Page save
-    if (props.formStateExists) {
-      setFormHasRendered(true);
-    }
```

Comment:
Why is this being removed? The comment makes it sound like it was important?

Reply:
The scenario for this being imporant is when clicking the page/parent save button and there was a validation issue with on the the child elements

This is no longer required as we're inline saving all elements and not sending any elemental data with the page/parent POST request